app/src/main/java/com/hedvig/app/feature/offer/ui/OfferModel.kt

```diff
@@ -28,13 +28,18 @@ sealed class OfferModel {
         object Coverage : Subheading()
     }
 
+    sealed class Paragraph : OfferModel() {
+        object Coverage: Paragraph()
+    }
+
     data class QuoteDetails(
         val name: String,
         val id: String,
     ) : OfferModel()
 
-    sealed class Paragraph : OfferModel() {
-        object Coverage: Paragraph()
-    }
+    data class FAQ(
+        val items: List<Pair<String, String>>
```

Comment:
Could perhaps refactor the Pair to a data class like 
```
data class FAQItem(
  val headLine: String,
  val body: String
)
```
Could make adding/removing fields easier later.

Reply:
Agreed.

I think I may actually have made a mental note to do that while implementing this feature, but of course, I forgot it. 😬